py-polars/polars/series/series.py

```diff
@@ -4523,6 +4523,12 @@ def to_pandas(  # noqa: D417
         3       4
         Name: b, dtype: int64[pyarrow]
         """
+        if self.dtype == Object:
+            # Can't convert via PyArrow, so do it via NumPy
+            series = pd.Series(self.to_numpy(), dtype=object)
+            series.name = self.name
+            return series
```

Comment:
Nitpicky but can we write this instead?

```suggestion
            return pd.Series(self.to_numpy(), dtype=object, name=self.name)
```